sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -761,12 +761,24 @@ booleanValue
     ;
 
 interval
-    : {ansi}? INTERVAL? intervalField+
-    | {!ansi}? INTERVAL intervalField*
+    : {ansi}? INTERVAL? (errorCapturingMultiUnitsInterval | errorCapturingUnitToUnitInterval)
+    | {!ansi}? INTERVAL (errorCapturingMultiUnitsInterval | errorCapturingUnitToUnitInterval)
```

Comment:
Just out of curiosity; is that bad to just drop `+` and `*`?
```
interval
    : {ansi}? INTERVAL? intervalField
    | {!ansi}? INTERVAL intervalField
```

Reply:
`intervalField` can be single-unit statements like `1 year`, which is allowed to repeat.

Reply:
Oh, I see. Thanks!